components/filesystem.rst

```diff
@@ -284,6 +284,8 @@ tempnam
 
     // returns a path like : /tmp/prefix_wyjgtF
     $filesystem->tempnam('/tmp', 'prefix_');
+    // returns a path like : /tmp/prefix_wyjgtF.png
+    $filesystem->tempnam('/tmp', 'prefix_', '.png');
```

Comment:
I think you can add a block below specifying the version this was introduced in

```
.. versionadded:: 4.4

    The option to set a suffix in  ``tempnam()`` was introduced in Symfony 4.4.
```

Reply:
Yes please 👍🏻